tests/utils_test.go

```diff
@@ -38,90 +39,97 @@ import (
+	testsSearchService                    *services.SearchService
+	testsDeleteService                    *services.DeleteService
+	testsDownloadService                  *services.DownloadService
+	testsSecurityService                  *services.SecurityService
+	testsCreateLocalRepositoryService     *services.LocalRepositoryService
+	testsCreateRemoteRepositoryService    *services.RemoteRepositoryService
+	testsCreateVirtualRepositoryService   *services.VirtualRepositoryService
+	testsCreateFederatedRepositoryService *services.FederatedRepositoryService
+	testsUpdateLocalRepositoryService     *services.LocalRepositoryService
+	testsUpdateRemoteRepositoryService    *services.RemoteRepositoryService
+	testsUpdateVirtualRepositoryService   *services.VirtualRepositoryService
+	testsUpdateFederatedRepositoryService *services.FederatedRepositoryService
+	testsDeleteRepositoryService          *services.DeleteRepositoryService
+	testsRepositoriesService              *services.RepositoriesService
+	testsCreateReplicationService         *services.CreateReplicationService
+	testsUpdateReplicationService         *services.UpdateReplicationService
+	testsReplicationGetService            *services.GetReplicationService
+	testsReplicationDeleteService         *services.DeleteReplicationService
+	testsPermissionTargetService          *services.PermissionTargetService
+	testUserService                       *services.UserService
+	testGroupService                      *services.GroupService
+	testBuildInfoService                  *services.BuildInfoService
+	testsFederationService                *services.FederationService
+
+	// Distribution services
+	testsBundleSetSigningKeyService      *distributionServices.SetSigningKeyService
+	testsBundleCreateService             *distributionServices.CreateReleaseBundleService
+	testsBundleUpdateService             *distributionServices.UpdateReleaseBundleService
+	testsBundleSignService               *distributionServices.SignBundleService
+	testsBundleDistributeService         *distributionServices.DistributeReleaseBundleService
+	testsBundleDistributionStatusService *distributionServices.DistributionStatusService
+	testsBundleDeleteLocalService        *distributionServices.DeleteLocalReleaseBundleService
+	testsBundleDeleteRemoteService       *distributionServices.DeleteReleaseBundleService
+
+	// Xray Services
+	testsXrayWatchService  *xrayServices.WatchService
+	testsXrayPolicyService *xrayServices.PolicyService
+	testXrayBinMgrService  *xrayServices.BinMgrService
+
+	// Pipelines Services
+	testsPipelinesIntegrationsService *pipelinesServices.IntegrationsService
+	testsPipelinesSourcesService      *pipelinesServices.SourcesService
+
+	// Access Services
+	testsAccessProjectService *accessServices.ProjectService
+
+	timestamp       = time.Now().Unix()
+	trueValue       = true
+	falseValue      = false
+	randomRunNumber = strings.Replace(uuid.NewString(), "-", "", -1)
```

Comment:
This is not a number. What do you think about `runUid`?
```suggestion
	runUid = strings.Replace(uuid.NewString(), "-", "", -1)
```